src/components/AskQuestion/AskQuestion.vue

```diff
@@ -140,6 +144,11 @@ export default {
   },
   methods: {
     validateQuestion() {
+      if (sessionStorage.getItem("userID") === null){
+        this.validUser = false;
+      }else {
+        this.validUser = true;
+      }
```

Comment:
validateQuestion validates a user? Please fix Naming of this function =)

Reply:
it validates every factor needed to send a question. and, the logged in state of a user is one part of it.. so i thought. 
but i will look for another solution